Sources/Winget-AutoUpdate/Winget-Install.ps1

```diff
@@ -202,43 +249,70 @@ function Install-App ($AppID, $AppArgs) {
                 Add-WAUWhiteList $AppID
             }
         }
-        else {
-            Write-ToLog "-> $AppID installation failed!" "Red"
-        }
     }
     else {
         Write-ToLog "-> $AppID is already installed." "Cyan"
     }
 }
 
-#Uninstall function
-function Uninstall-App ($AppID, $AppArgs) {
+# Uninstall function
+function Uninstall-App ($AppID) {
```

Comment:
What happens with the Uninstall with $AppArgs from my commit https://github.com/Romanitho/Winget-AutoUpdate/commit/22183bcb0e153cedb147f7b90b5d52eed3f83298 ?

Reply:
Sorry, missed that. Now fixed